static/templates/settings/display_settings.hbs

```diff
@@ -29,7 +29,7 @@
     </div>
 
 
-    <div class="theme-settings {{#if for_realm_settings}}org-subsection-parent{{else}}subsection-parent{{/if}}">
+    <div class="theme-settings {{#if for_realm_settings}}settings-subsection-parent{{else}}subsection-parent{{/if}}">
```

Comment:
Hmm, I'm not sure about this name; `settings-subsection-parent` vs. `subsection-parent` doesn't communicate clearly the "Has a save/discard widget or not". Perhaps `settings-discardable-subsection-parent` vs. `settings-autosave-subsection-parent` would be appropriate? I guess I'm not sure where we currently use `subsection-parent` itself...

I guess I don't understand why `display_settings` ends up having an if/else when things like "chsto

Reply:
Sounds reasonable.